apps/coscrad-frontend/src/components/resources/transcribed-audio/transcribed-audio-detail.full-view.presenter.tsx

```diff
@@ -0,0 +1,31 @@
+import { IDetailQueryResult, ITranscribedAudioViewModel } from '@coscrad/api-interfaces';
+import { MediaPlayer } from '@coscrad/media-player';
+import './transcribed-audio-detail.full-view.presenter.css';
+
+export const TranscribedAudioDetailFullViewPresenter = ({
+    data: transcribedAudioItem,
+}: IDetailQueryResult<ITranscribedAudioViewModel>): JSX.Element => {
+    const runTime = Math.round(transcribedAudioItem.length / 1000);
+
+    return (
```

Comment:
Destructure above and use an implicit return.

Reply:
Not sure how this would work if I need the run time in seconds?

Reply:
Or are you saying to stick with milliseconds for now as in the current model but then change the view model to have seconds?  I'm just trying to make this human readable in the presenter.  However it's important to note that the player itself will need milliseconds for more precise indexing.

Reply:
destructuring complete